scss/styles.scss

```diff
@@ -0,0 +1,3 @@
+body {
+  padding: 32px;
```

Comment:
Are you sure you need this in the body? Maybe the view container is a better suit 😄

Reply:
Fixed in PR #6